assets/component-slider.css

```diff
@@ -136,6 +149,13 @@ slider-component.slider-component-full-width {
     scroll-padding-left: var(--desktop-margin-left-first-item);
   }
 
+  .slider--desktop:after {
```

Comment:
A little redundant but I think it's worth duplicating the comment for each of these main `slider--mobile/tablet/desktop:after` instances. Or else group them together below a common comment. Lot of this stuff is shared between them anyway.

This sort of unidiomatic workaround is a shining example of the kind of thing where comments are 100% justified as far as I'm concerned.